packages/schema/tests/generator/prisma-generator.test.ts

```diff
@@ -328,4 +332,34 @@ describe('Prisma generator test', () => {
         const todo = dmmf.datamodel.models.find((m) => m.name === 'Todo');
         expect(todo?.documentation?.replace(/\s/g, '')).toBe(`@@allow('read', owner == auth())`.replace(/\s/g, ''));
     });
+
+    it('format prisma', async () => {
+        const model = await loadModel(`
+            datasource db {
+                provider = 'postgresql'
+                url = env('URL')
+            }
+
+            model Post {
+                id Int @id() @default(autoincrement())
+                title String
+                content String?
+                published Boolean @default(false)
+                @@allow('read', published)
```

Comment:
There should be a blank line before published to verify that the lines inside models are preserved.